atomics/T1105/T1105.yaml

```diff
@@ -298,4 +298,29 @@ atomic_tests:
       del C:\\svchost.exe >nul 2>&1
     name: command_prompt
     elevation_required: true
-
+- name: MpCmdRun DownloadFile
+  description: LOLB to download a file using Windows Defender that will put Atomic-license.txt in temp directory using this version 4.18.2007.8-0
```

Comment:
Can you clarify that the version being referred to here is the Windows Defender version and how users can check their version? Also a link to a related blog post with more information would be great if you can find one.